### PR TITLE
Fix root route mapping

### PR DESCRIPTION
**Pull request by codetheweb**

### CODE REVIEW

File: src/routes/create-route-map-from-directory.ts [hunk 1]
```diff
@@ -26,5 +26,11 @@ export const createRouteMapFromDirectory = async (
     }
   }
 
+  // Fix root route
+  if (routes[""]) {
+    routes["/"] = routes[""]
+    delete routes[""]
+  }
```
Comment (mxsdev) [inline src/routes/create-route-map-from-directory.ts:33]: how does this happen? shouldn't `routeWithSlash` prevent this?

Comment (codetheweb) [reply src/routes/create-route-map-from-directory.ts:33]: no, because it replaces `/*` with `""`

Comment (mxsdev) [reply src/routes/create-route-map-from-directory.ts:33]: is there a more elegant way to handle this?

Comment (codetheweb) [reply src/routes/create-route-map-from-directory.ts:33]: you could probably get clever with the regex so it doesn't capture the `/` when it's `/index.ts` (vs `/foo/index.ts`) but imo the current solution is probably a bit more readable than that

Comment (mxsdev) [reply src/routes/create-route-map-from-directory.ts:33]: K

**Review by mxsdev — APPROVED** [review]

**Pull request merged**